### PR TITLE
RF: Flatten the window folder to the file

### PR DESCRIPTION
**Pull request by maharshi-gor**
### RF: Flatten the window folder to the file

**PR Contents**
- Flatten the window folder to the file.
- Added all the pygfx imports from the window file to the `lib.py`.
- Moved `jupyter-rfb` as an optional dependency under the plot.

### CODE REVIEW

**Comment by maharshi-gor** [comment]
Hey @skoudoro , I have not added the docstring intentionally as It will change a lot in the upcoming PRs.

I have also figured out some tests, It will create a change in a lot of files and a lot of places. I am breaking this big change in to small changes, that are easy to review.

PS: This also tackles the issue of jupyter-rfb not being the optional dependency.

File: fury/lib.py [hunk 1]
```diff
@@ -1,14 +1,42 @@
...
 Background = gfx.Background
 BackgroundSkyboxMaterial = gfx.BackgroundSkyboxMaterial
-Camera = gfx.Camera
-Controller = gfx.Controller
+
+# Classes that needed to be written as types
+Camera: TypeAlias = gfx.Camera
+Controller: TypeAlias = gfx.Controller
+Scene: TypeAlias = gfx.Scene
+Viewport: TypeAlias = gfx.Viewport
```
Comment (skoudoro) [inline fury/lib.py:30]: What do you need the TypeAlias here ?

Comment (maharshi-gor) [reply fury/lib.py:30]: It gives pre-commit error as these classes are used to either extend the class or used as type in a dataclass.

Comment (skoudoro) [reply fury/lib.py:30]: strange, ok, I look into it later, merging

**Review by skoudoro — COMMENTED** [review]
all ok, merging

File: fury/lib.py (hunk 1 above)
Comment (maharshi-gor) [reply fury/lib.py:30]: @skoudoro  you can take a look at this. The pre-commit points to this resource when it gives the error. https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases

**Pull request merged**